src/util/initEvents.ts

```diff
@@ -227,6 +227,7 @@ const initEvents = (store: Store<State, any>) => {
   }
 
   /** Stops the scroll-at-edge when dragging stops. */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const onTouchEnd = (e: TouchEvent) => {
```

Comment:
@raineorshine 

here e is unused argument. if i remove this then i have error in other file as well where we are calling onTouchEnd function so should i remove e as argument from all the files or just set the command line here to ignore the lint error?

Reply:
there are similar type of functions in code with same above error. so i will apply your answer on all these type of same functions. like canDrop onDrag

Reply:
The solution is to explicitly type the function, then omit the parameter from the definition:

```ts
  /** Stops the scroll-at-edge when dragging stops. */
  const onTouchEnd: (e: TouchEvent) => void = () => {
    scrollAtEdge.stop()
  }
```

Reply:
@raineorshine 
you mean i should remove the parameter from definition and also from all the calling occurrences.

Reply:
No, you shouldn't have to change the callers. The function should be able to be called with an event object, but it doesn't have to use it in the body of the function definition. As long as the function is explicitly typed to take the event object, it can be omitted from the definition (as seen above).

Reply:
Yes please demonstrate, there is confusion.

Reply:
Sure, I can demonstrate it in a separate commit. However, the build is currently broken and the branch is falling behind. Once you have an up-to-date branch with a working build and passing CI again I can add to it.